utils.go

```diff
@@ -33,6 +34,7 @@ func (s *Service) String() string {
 
 // Storage is the s3 object storage service.
 type Storage struct {
+	typ.UnimplementedStorager
```

Comment:
ditto.

And s3 implement following features: https://github.com/aos-dev/go-service-s3/blob/master/service.toml#L19

So we also need to add `typ.UnimplementedMultiparter`.